os/pm/Kconfig

```diff
@@ -42,6 +39,63 @@ config PM_METRICS_DURATION
 
 endif
 
+comment "PM Options"
+
+config PM_NSTATE
+	int "Number of SoC sleep states"
+	default ""
+	---help---
+		Number of sleep states provided by the SoC Power module
+
+if PM_NSTATE != 0
+config PM_STATE_SWITCH_THRESH
+	string "PM STATE count threshold"
+	default ""
+	---help---
+		Enter values in the form: value_1,value_2,
+
+		State changes occur when the activity accumulator count crosses
+		threshold values for certain periods of time (time slice count).
+
+		CONFIG_PM_STATE_SWITCH_THRESH is the count for entering state xxx.
```

Comment:
CONFIG_PM_STATE_SWITCH_THRESH is the **time slice** count for entering state xxx.